etc/config/swift.amazon.properties

```diff
@@ -24,16 +36,22 @@ compiler.swiftdevsnapshot.semver=devsnapshot
 # Release Compilers
 #################################
 
-# 6.x
+# 6.x (x86)
 
 compiler.swift603.exe=/opt/compiler-explorer/swift-6.0.3/usr/bin/swiftc
 compiler.swift603.semver=6.0.3
 
+# 6.x (arm)
+
+compiler.arm-swift603.exe=/opt/compiler-explorer/swift-6.0.3/usr/bin/swiftc
+compiler.arm-swift603.semver=6.0.3
+compiler.arm-swift603.options=-target aarch64-swift-linux-musl -sdk /opt/compiler-explorer/swift-6.0.3-static-sdk/swift-linux-musl/musl-1.2.5.sdk/aarch64 -resource-dir /opt/compiler-explorer/swift-6.0.3-static-sdk/swift-linux-musl/musl-1.2.5.sdk/aarch64/usr/lib/swift_static
```

Comment:
you can test these additional options applied to the existing 6.0.3 x86 compiler here: https://swift.godbolt.org/z/8W997od7q

Reply:
Thanks!